string type
int32[] img_resolution
string[] label_list

int32[] mask_shape
float32[] mask

int32[] boxes_shape
float32[] boxes

int32[] scores_shape
float32[] scores

int32[] labels_shape
int32[] labels